core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreUtil.java

```diff
@@ -111,4 +117,53 @@ public static Certificate getCertificate(String alias, KeyStore store) throws Ax
         }
     }
 
+    public static boolean isCustomKeyStore(String keyStoreName) {
```

Comment:
Add javadoc comment for the method

Reply:
Fixed in https://github.com/wso2/carbon-kernel/commit/a0533c76776fd427f270477373ea6951ce2f147f